ale_linters/ada/adalsp.vim

```diff
@@ -0,0 +1,16 @@
+" Author: Bartek Jasicki http://github.com/thindil
+" Description: Support for Ada Language Server
+
+call ale#Set('ada_lsp_executable', 'ada_language_server')
+
+function! ale_linters#ada#adalsp#GetRootDirectory(buffer) abort
+    return fnamemodify(bufname(a:buffer), ':p:h')
+endfunction
+
+call ale#linter#Define('ada', {
+\   'name': 'adalsp',
```

Comment:
Let's name this `adals`, and name the variable aboev `ada_adals_executable`. That will match some existing linter names like `dls` for D, and how the options are named for that.